tracer/src/Datadog.Trace/Telemetry/Collectors/ProductsTelemetryCollector.cs

```diff
@@ -52,16 +53,19 @@ public void ProductChanged(TelemetryProductType product, bool enabled, ErrorData
             var appsec = GetLatestData(_allTime, _productsByType, TelemetryProductType.AppSec);
             var profiler = GetLatestData(_allTime, _productsByType, TelemetryProductType.Profiler);
             var dynamicInstrumentation = GetLatestData(_allTime, _productsByType, TelemetryProductType.DynamicInstrumentation);
+            var exceptionDebugging = GetLatestData(_allTime, _productsByType, TelemetryProductType.ExceptionDebugging);
```

Comment:
Thanks for sharing this! Yes, in our internal talks we consider the Exception Debugging a different product in the meaning that there should not be reliance on the Dynamic Instrumentation enablement when the customer is only interested in Exception Debugging. The underlying implementation indeed relies on Probes and DI instrumentation but that's an implementation detail. In the eyes of the customer he/she should enable it separately.

I will look into the telemetry thing.

Reply:
For simplicity I've removed Exception Debugging from telemetry in aa0a6382d93ee68d3bf1565011bd823df3df9917. It will be reconsidered later if it should be part of this reporting, or as part of the config sent to our backend as part of bootstrapping.